core/src/main/java/com/linecorp/armeria/common/logging/ContentPreviewer.java

```diff
@@ -86,6 +86,13 @@ static ContentPreviewer disabled() {
         return ContentPreviewerAdapter.NOOP;
     }
 
+    /**
+     * Returns whether this {@link ContentPreviewer} is {@link #disabled()} or not.
+      */
+    default boolean isDisabled() {
+        return this == ContentPreviewerAdapter.NOOP;
```

Comment:
I think it's more idiomatic to `return false` here and have `NOOP` override with `return true`

Reply:
Fixed. 😉